### PR TITLE
feat(cb2-3354): viewing type approval section for psv tech record (#317)

### PR DESCRIPTION
**Pull request by premghinde**
## CB2-3384 - Viewing type approval section for psv tech record

Adding body type block to tech record
[CB2-3384](https://dvsa.atlassian.net/browse/CB2-3384)

## Checklist

- [ / ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ / ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ / ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/features/tech-record-summary/tech-record-summary.component.html [hunk 1]
```diff
@@ -8,4 +8,5 @@
   <app-dynamic-form-group *ngIf="applicantDetailsTemplate" [template]="applicantDetailsTemplate" [data]="currentRecord" class="govuk-summary-list"></app-dynamic-form-group>
   <app-dynamic-form-group *ngIf="documentsTemplate" [template]="documentsTemplate" [data]="currentRecord" class="govuk-summary-list"></app-dynamic-form-group>
   <app-dynamic-form-group *ngIf="notesTemplate" [template]="notesTemplate" [data]="currentRecord" class="govuk-summary-list"></app-dynamic-form-group>
+  <app-dynamic-form-group *ngIf="body" [template]="body" [data]="currentRecord" class="govuk-summary-list"></app-dynamic-form-group>
```
Comment (naathanbrown) [inline src/app/features/tech-record-summary/tech-record-summary.component.html:11]: can we rename this bodyTemplate please?

File: src/app/forms/templates/psv/psv-tech-record.template.ts [hunk 2]
```diff
@@ -205,6 +205,52 @@ export const PsvTechRecord: FormNode = {
...
+    },
+    {
+      name: 'variantNumber',
+      label: 'Variant number',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'variantVersionNumber',
+      label: 'Variant version number',
+      type: FormNodeTypes.CONTROL
```
Comment (naathanbrown) [inline src/app/forms/templates/psv/psv-tech-record.template.ts:253]: Isn't this the tyre approval section? I think on develop this already exists as it's own template?

Comment (premghinde) [reply src/app/forms/templates/psv/psv-tech-record.template.ts:253]: Don't know why this came through in the rebase. I didn't do this work.

Comment (naathanbrown) [reply src/app/forms/templates/psv/psv-tech-record.template.ts:253]: Yeah odd it's come through, I think it was like this by mistake and then we removed it in a PR afterwards, so maybe you rebased before that change and just need to do it again?

Comment (premghinde) [reply src/app/forms/templates/psv/psv-tech-record.template.ts:253]: Re-rebased and it's still here :(

Comment (premghinde) [reply src/app/forms/templates/psv/psv-tech-record.template.ts]: Code deleted @naathanbrown

**Review by naathanbrown — APPROVED** [review]

**Review by kpearmanbjss — APPROVED** [review]

**Review by danielchildsbjss — APPROVED** [review]

**Review by gjulien-bjss — APPROVED** [review]

**Pull request merged**